graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/journal/RawMessage.java

```diff
@@ -166,7 +167,7 @@ public int getVersion() {
     }
 
     public DateTime getTimestamp() {
-        return new DateTime(msgBuilder.getTimestamp()); // TODO PERFORMANCE object creation
+        return new DateTime(msgBuilder.getTimestamp(), DateTimeZone.UTC); // TODO PERFORMANCE object creation
```

Comment:
Can this cause problems on the reading side of the journal? Just changing it from the local TZ to UTC might have consequences.